src/utils/chart.ts

```diff
@@ -0,0 +1,120 @@
+import { graphic } from "echarts/core";
+
+function formatter(value: Number) {
+  return value >= 100000 ? value.toExponential() : value;
+}
```

Comment:
Return type

```suggestion
function formatter(value: Number): Number {
  return value >= 100000 ? value.toExponential() : value;
}
```

(is my syntax correct?)

Reply:
Fixed, it's actually a string